.travis.yml

```diff
@@ -14,14 +14,12 @@ env:
 matrix:
     fast_finish: true
     include:
-        - php: 7.1
         - php: 7.2
         - php: 7.3
           env: deps=low
 
 before_install:
     - phpenv config-rm xdebug.ini || true
-    - composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-master
```

Comment:
changing `symfony/flex dev-master` to `symfony/flex dev-main` will fix the Travis error "...cant find dev-master...."

Symfony renamed all of the "master" branches to "main" a few weeks ago..

Reply:
Yes, I only dropped this because I assumed there would not be any benefit compared to composer v2 but I can readd it with the changed branch name of course.

Reply:
reverted + changed in dbb215ee8fced48e6803859f891dda40f18666f8